exampleCourse/questions/element/integerInput/question.html

```diff
@@ -21,11 +21,12 @@
         ><code>pl-integer-input</code></a
       >
       element. In particular, the default input field has help text of "integer" as a placeholder
-      as well as a question button.
+      as well as a question button. The attribute <code>initial-value</code> pre-fills the input box
+      the first time it is rendered so students can start from a suggested value.
```

Comment:
Probably not a good idea to use an example that makes reference to the placeholder as the starting point.